test/PolymerDom_test.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+  <head>
```

Comment:
works now, I needed to wait for polymer to be ready, but I couldn't get it working with the webcomponents package, I did now some (little bit ugly) js interop